app/workers/aggregations_dump_worker.rb

```diff
@@ -1,29 +1,9 @@
 class AggregationsDumpWorker
   include Sidekiq::Worker
-  include DumpWorker
-  include RateLimitDumpWorker
 
   sidekiq_options queue: :data_high
 
-  def perform_dump
-    raise ApiErrors::FeatureDisabled unless Panoptes.flipper[:dump_worker_exports].enabled?
-    # Set expiry time of signed_url to one day from now
-    medium.update!(put_expires: 1.day.from_now.to_i - Time.now.to_i)
-
-    AggregationClient.new.aggregate(resource, medium)
-  end
-
-  def upload_dump
-    # aggregation engine will write directly to medium when done, no need to upload
-  end
-
-  def cleanup_dump
-    # aggregation engine will write directly to medium when done, no need to cleanup
-  end
-
-  def load_medium
-    m = Medium.find(@medium_id)
-    m.update!(path_opts: resource_file_path, private: true, content_type: "application/x-gzip")
-    m
+  def perform(resource_id, resource_type, medium_id=nil, requester_id=nil, *args)
+    raise "No longer supported, nobody should be triggering this."
```

Comment:
Last one was generated months ago. Button in PFE is gone afaik. @camallen happy with this change?

Reply:
:tada: So happy